ios/AudioInputController.m

```diff
@@ -146,13 +161,13 @@ - (OSStatus) _initializeAudioGraph {
     if (_checkError(status, "Couldn't set RemoteIO's render callback on bus 0")) {
         return status;
     }
-    
```

Comment:
do these whitespace changes mean that you found a formatter for objective-c?

Reply:
`CMD + A` then `CTRL + I` in XCode